sparse/usr/sbin/flash-partition

```diff
@@ -199,6 +199,10 @@ if on_device; then
                 fi
         fi
 
+	if [[ "$SAILFISHOS_WIPE_PARTITIONS" == "1" ]]; then
+		dd if=/dev/zero bs=4096 of=$DEVDIR/$DEVICE
```

Comment:
The dd immediately after this one was 4096.  These are the small partitions btw.  The lvm/btfs partition is wiped here https://github.com/sailfishos/initrd-helpers/pull/11/files and that was 1M before I changed it be consistent with this.  Interestingly I saw no difference in the time it took.